ext/configuration.h

```diff
@@ -128,8 +128,6 @@ extern bool runtime_config_first_init;
     CONFIG(BOOL, DD_TRACE_RETAIN_THREAD_CAPABILITIES, "false", .ini_change = zai_config_system_ini_change)     \
     CONFIG(STRING, DD_VERSION, "")                                                                             \
     CONFIG(STRING, DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP, DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP_DEFAULT) \
-    CONFIG(BOOL, DD_TRACE_CLIENT_IP_HEADER_DISABLED, "false")                                                  \
```

Comment:
If you go on with this change, please update the public doc as well as it's documented [there](https://docs.datadoghq.com/tracing/trace_collection/library_config/php/#environment-variable-configuration)

Reply:
Thanks for the heads up! I will change it

Reply:
https://github.com/DataDog/documentation/pull/15697